rag/tests/unit/test_retrieval_service.py

```
@@ -52,92 +52,91 @@ def service(mock_embedder, mock_vector_repo, mock_resolver):
 
-        service.search(query="test", limit=5, scope="private", user="alice")
+        service.search(query="test", owner_id="bob", limit=5)
 
         call_kwargs = mock_vector_repo.search.call_args
         filters = call_kwargs.kwargs.get("filters") or call_kwargs[1].get("filters")
-        assert filters["metadata.upload_by"] == "alice"
+        assert filters["metadata.owner_id"] == "bob"
 
-    def test_search_public_scope(self, service, mock_resolver, mock_vector_repo):
-        """Public scope must not add an upload_by filter, returning results from all users.
+    def test_search_resolver_receives_owner_id(self, service, mock_resolver, mock_vector_repo):
```

Ruff (0.15.20)

[warning] Unused method argument: `mock_vector_repo`

(ARG002)